IDE/Renesas/e2studio/RX72N/EnvisionKit_Toppers/WolfSSLDemo/src/wolfDemo/wolfssl_dummy.c

```diff
@@ -0,0 +1,61 @@
+/* wolfssl_dummy.c
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+#ifdef HAVE_CONFIG_H
+    #include <wolfssl/src/config.h>
+#endif
+
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfssl/ssl.h>
+#include <wolfssl/wolfcrypt/types.h>
+#include <wolfssl/wolfcrypt/error-crypt.h>
+#include <stdio.h>
+
+#define YEAR 2022
+#define MON  9
```

Comment:
Please change YEAR and MON so that it is able to load the latest certs.
```
The latest ca-cert.pem's Not Before is:
  Dec 16 21:17:49 2022 GMT

e.g.
  YEAR 2023
  MON  1
```

Reply:
Update